<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIEquipSettings</class>
 <widget class="QWidget" name="WIEquipSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>427</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="equipSettingsLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <property name="leftMargin">
      <number>6</number>
     </property>
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="rightMargin">
      <number>6</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QCheckBox" name="kcfg_NoEquipCheck">
       <property name="text">
        <string>No equipment</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="kcfg_TelescopeCheck">
       <property name="text">
        <string>Telescope</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scopeSelectLabel">
       <property name="text">
        <string>Select telescope from list to use or specify new telescope</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="scopeFrame">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>5</number>
        </property>
        <item>
         <widget class="QListWidget" name="scopeListWidget">
          <property name="minimumSize">
           <size>
            <width>160</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="scopeDetailsBox">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>0</height>
           </size>
          </property>
          <property name="title">
           <string>Telescope Details</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <property name="bottomMargin">
            <number>50</number>
           </property>
           <item row="4" column="1">
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="5" column="2">
            <widget class="QLabel" name="apertureText">
             <property name="text">
              <string>--        </string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="3" column="2">
            <widget class="QLabel" name="modelText">
             <property name="text">
              <string>--        </string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="vendorText">
             <property name="text">
              <string>--        </string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="scopeModelText">
             <property name="text">
              <string>Model:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="scopeVendorText">
             <property name="text">
              <string>Vendor: </string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="scopeApertureText">
             <property name="text">
              <string>Aperture:</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="addScopeFrame">
          <property name="minimumSize">
           <size>
            <width>220</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>194</width>
            <height>16777215</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <spacer name="verticalSpacer_9">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="newScopeLabel">
             <property name="text">
              <string>Add new telescope:</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_6">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="vendorLabel">
               <property name="text">
                <string>Vendor: </string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="vendorLineEdit"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="modelLabel">
               <property name="text">
                <string>Model: </string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="modelLineEdit"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="apertureLabel">
               <property name="text">
                <string>Aperture: </string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QDoubleSpinBox" name="apertureSpinBox">
                 <property name="maximum">
                  <double>9999.000000000000000</double>
                 </property>
                 <property name="singleStep">
                  <double>0.100000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="focalLengthLabel">
               <property name="text">
                <string>Focal Length: </string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QDoubleSpinBox" name="focalLenSpinBox">
                 <property name="maximum">
                  <double>9999.989999999999782</double>
                 </property>
                 <property name="singleStep">
                  <double>0.010000000000000</double>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>mm</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="typeLabel">
               <property name="text">
                <string>Type</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QComboBox" name="typeComboBox">
               <item>
                <property name="text">
                 <string>Reflector</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Refractor</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="driverLabel">
               <property name="text">
                <string>Driver:</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QComboBox" name="driverComboBox">
               <item>
                <property name="text">
                 <string>None</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_7">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="saveNewScopeButton">
             <property name="text">
              <string>Save new telescope</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer_8">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="binoGroupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QCheckBox" name="kcfg_BinocularsCheck">
          <property name="text">
           <string>Binoculars</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="binoDetailsFrame">
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="binoApertureText">
             <property name="text">
              <string>Specify aperture:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="kcfg_BinocularsAperture">
             <property name="value">
              <double>40.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="binoApertureUnit">
             <property name="text">
              <string>mm</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
